base_layer/core/src/transactions/transaction.rs

```diff
@@ -399,7 +399,7 @@ impl Display for TransactionOutput {
     fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), std::fmt::Error> {
         let proof = self.proof.to_hex();
         fmt.write_str(&format!(
-            "{} [{:?}] Proof: {}..{}\n",
+            "({} [{:?}] Proof: {}..{})",
```

Comment:
see review comment

Reply:
Agreed, this should not have been changed.

Reply:
Updated the PR description.